services/app-api/src/errors/errors.ts

```diff
@@ -0,0 +1,9 @@
+class NotFoundError extends Error {
```

Comment:
I think this error can live inside the postgres package and be exported from there, I see it as part of the contract of that package, since some of those functions can return it. 

You may ask, what makes this different from a domain model then, and I don't have a great answer, except that I don't see us reusing the not found error for other packages? I could be wrong.

Reply:
That sounds good to me!